pages/[review].js

```diff
@@ -0,0 +1,103 @@
+import React, { useState } from "react";
+import Link from "next/link";
+import { globalReviews } from ".";
+import { useAtom } from "jotai";
+import { useRouter } from "next/router";
+
+console.clear();
+
+const initialAspects = [
+  { id: "1", name: "Hospitality", status: false },
+  { id: "2", name: "Cleanness", status: false },
+  { id: "3", name: "Food Quality", status: false },
+  { id: "4", name: "Food Quantity", status: false },
+  { id: "5", name: "Ambience", status: false },
+  { id: "6", name: "Friendliness", status: false },
+];
+
+export default function GoodPage() {
+  const router = useRouter();
+  const [aspects, setAspects] = useState(initialAspects);
+  const [reviews, setReviews] = useAtom(globalReviews);
+  const { review } = router.query;
+
+  if (review !== "good" && review !== "neutral" && review !== "bad") {
```

Comment:
Simple and effective. 👍

Reply:
instead of this triple AND operation you could use the .includes method:
```js
if( !['good','neutral','bad'].includes(review) ){ ... }
```